src/ng/compile.js

```diff
@@ -1671,7 +1671,8 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         nodeName = nodeName_(this.$$element);
 
         if ((nodeName === 'a' && (key === 'href' || key === 'xlinkHref')) ||
-            (nodeName === 'img' && key === 'src')) {
+            (nodeName === 'img' && key === 'src') ||
+            (nodeName === 'image' && key === 'xlinkHref')) {
           // sanitize a[href] and img[src] values
           this[key] = value = $$sanitizeUri(value, key === 'src');
```

Comment:
Shouldn't `key ===src` be updated to also account for `<image>` elements. Shouldn't `image[xlink:href]` be sanitized based on `imgSrcSanitizationWhitelist` rather than `aHrefSanitizationWhitelist`?